src/main/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelper.java

```diff
@@ -123,7 +123,7 @@ private boolean shouldNotUseProxy(String url) {
     }
 
     private boolean shouldIgnoreHost(String hostToMatch) {
-        if (StringUtils.isBlank(hostToMatch) || ignoredProxyHosts == null || ignoredProxyHosts.isEmpty()) {
+        if (ignoredProxyHosts == null || ignoredProxyHosts.isEmpty()) {
```

Comment:
shouldIgnoreHost() is only called by shouldNotUseProxy(). If the input to shouldNotUseProxy() is null or an empty String, new URL(url) will throw MalformedURLException, which is handled, and shouldIgnoreHost() is never called. The condition removed here can never happen.